2023/1002xxx/GSD-2023-1002425.json

```diff
@@ -51,10 +51,11 @@
       ],
       "references": [
         {
-          "type": "WEB",
+          "type": "FIX",
           "url": "https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=94f68f3e059c478e240f65fcb64746fe371295df"
         }
-      ]
+      ],
+      "aliases": []
```

Comment:
I could make it so if it's not populated yet and the new value is also empty, it leaves it untouched...Not sure how much we care, as explicitly stating there are no aliases isn't a bad thing.